src/Algebra/Graph/Undirected.hs

```diff
@@ -698,6 +700,25 @@ toSymmetricRelation :: Ord a => Graph a -> SR.Relation a
 toSymmetricRelation = foldg SR.empty SR.vertex SR.overlay SR.connect
 {-# INLINE toSymmetricRelation #-}
 
+-- | Complement of a graph.
+-- Complexity: /O(E+V)/ time, /O(E+V)/ memory where
+-- E is the number of edges and V is the number of vertices.
+--
+-- @
+-- complement 'empty'           == 'empty'
+-- complement ('vertex' x)      == ('vertex' x)
+-- complement ('edge' x y)      == ('overlay' ('vertex' x) ('vertex' y))
+-- complement ('star' x [y, z]) == ('overlay' ('vertex' x) ('edge' y z))
+-- complement . complement      == id
```

Comment:
I suspect this line will render misaligned in Haddock. Could you build the docs and double-check?